tt_metal/tools/profiler/profile_this.py

```diff
@@ -26,7 +26,7 @@ def profile_command(test_command, output_folder, name_append):
         options += f"-o {output_folder}"
     if name_append:
         options += f" -n {name_append}"
-    opProfilerTestCommand = f"python -m tracy -r -p {options} -m {test_command}"
+    opProfilerTestCommand = f"python -m tracy -v -r -p {options} -m {test_command}"
```

Comment:
@mo-tenstorrent , are you ok with adding "-v" here by default?

Reply:
Let's see if folks will start complaining about their run logs getting crowded 😄